.pipelines/scripts/run-e2e.sh

```diff
@@ -145,6 +146,7 @@ fi
 
 export E2E_ON_AKS_CLUSTER=true
 if [[ "${CLUSTER_TYPE:-}" =~ "autoscaling" ]]; then
-  export LABEL_FILTER="Feature:Autoscaling || !Serial && !Slow"
+
+  export LABEL_FILTER="${LABEL_FILTER:-"Feature:Autoscaling || !Serial && !Slow"}"
```

Comment:
Is the quotation mark usage correct here?

Reply:
I have removed the quotation mark, and corrected it. To ensure it's working correctly, 
I tried rerunning the pipeline here:
https://dev.azure.com/AzureContainerUpstream/Kubernetes/_build/results?buildId=72939&view=results
(test may not be completed by the time you review this, but correct tests are running based on that filter)